src/test/resources/translator/language_no_dup_key.xsd

```diff
@@ -21,7 +21,7 @@
                     <xs:complexType>
                         <xs:sequence>
                             <xs:element type="stringNotEmpty" name="key"/>
-                            <xs:element type="stringNotEmpty" name="value"/>
+                            <xs:element type="xs:string" name="value"/><!-- type from "stringNotEmpty" changed to "xs:string" to remove restriction to allow empty value -->
```

Comment:
I don't think the comment is necessary, it just described what is in the code, not why.